### PR TITLE
Added GET to return all tasks from a story

### PR DESCRIPTION
**Pull request by Bruno-Israel-Mendoza-Aguilar**
Added task model, domain, custom repository and respective methods

### CODE REVIEW

File: src/main/java/com/stories/controller/StoriesController.java [hunk 1]
```diff
@@ -79,4 +81,13 @@ public void deleteStory(@Valid @PathVariable String id) throws Exception {
 	public StoryDomain updateStory(@Valid @RequestBody StoryDomain request, @PathVariable String id) throws Exception {
 		return storyService.updateStory(request, id);
 	}
+	
+		@ApiOperation(value = " GET Tasks ", notes = " This operation will return the tasks of a story ")
+		@ApiResponses({ @ApiResponse(code = 200, message = " Success operation ")})
+		@ResponseStatus(value = HttpStatus.OK)
+		@GetMapping(value = "/{id}/tasks", produces = "application/json")
+		@ResponseBody
+		public List<TasksDomain> findTasksByyId(@Valid @PathVariable String id) throws EntityNotFoundException{
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/controller/StoriesController.java]: We could follow the standar of the names of storyAPI, could you please change the name like: getTasksByStory ?

File: src/main/java/com/stories/model/TaskModel.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.Data;
+
+@Data
+@Document(collection = "stories")
+public class TaskModel {
+
+	private String _id;
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/model/TaskModel.java:13]: Is not necessary add a @ID for mongoDB ?

File: src/main/java/com/stories/service/StoriesServiceImpl.java [hunk 3]
```diff
@@ -116,6 +123,20 @@ public StoryDomain getStoryById(String id) throws Exception {
 		logger.debug("Getting all stories - JSON : {}", allStoriesDomain);
 		return allStoriesDomain;
 	}
+	
+	public List<TasksDomain> findTasksByStoryId(String id) throws EntityNotFoundException{
+		if(storiesRepository.existsById(id)) {
+			Aggregation aggregation = Aggregation.newAggregation(
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/service/StoriesServiceImpl.java]: Please lest follow the standars ,this Aggretion objet should be on mongo Repository

**Review by VictorMunguiaEnciso — CHANGES_REQUESTED** [review]

File: src/main/java/com/stories/model/TaskModel.java (hunk 2 above)
Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/java/com/stories/model/TaskModel.java:13]: fixed

File: src/main/java/com/stories/controller/StoriesController.java (hunk 1 above)
Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/java/com/stories/controller/StoriesController.java]: fixed

File: src/main/java/com/stories/service/StoriesServiceImpl.java (hunk 3 above)
Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/java/com/stories/service/StoriesServiceImpl.java]: fixed

**Review by VictorMunguiaEnciso — DISMISSED** [review]

File: src/main/java/com/stories/domain/StoryDomain.java [hunk 4]
```diff
@@ -60,8 +60,12 @@
 	
 	@ApiModelProperty(example="1", value="The history of the story")
 	private List<String> history;
+	
+	@ApiModelProperty(example="?", value="The tasks of the story")
```
Comment (soydianapinto) [inline src/main/java/com/stories/domain/StoryDomain.java]: What do you mean with this character "?"?

Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/java/com/stories/domain/StoryDomain.java]: Sorry, I forgot to change that. As for the example, I see on other annotations from the same class that the value there is just "1", should I change it to that?

Comment (soydianapinto) [reply src/main/java/com/stories/domain/StoryDomain.java]: You can change it for "New task"

Comment (Bruno-Israel-Mendoza-Aguilar) [reply src/main/java/com/stories/domain/StoryDomain.java]: Thank you, it´s been already changed

**Review by soydianapinto — APPROVED** [review]

**Pull request merged**